README.md

```diff
@@ -1,6 +1,6 @@
 # Dive into Machine Learning [![Creative Commons License](http://i.creativecommons.org/l/by/4.0/88x31.png)](https://creativecommons.org/licenses/by/4.0/) [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
-Hi there! This guide is for you if:
+This is a hack-first guide that you might find particularly useful if ...
```

Comment:
"hack" as in "hands-on" and "making" ... But I realized, someone might mistake the meaning. It's too idiomatic/specific of a usage to make sense right up here at the intro. So, nevermind